src/sentry/static/sentry/app/components/deployBadge.tsx

```diff
@@ -12,12 +12,13 @@ import overflowEllipsis from 'app/styles/overflowEllipsis';
 
 type Props = {
   deploy: Deploy;
+  projectId: number;
```

Comment:
This should be optional because sometimes this deployBadge is used without being a clickable link.

Reply:
yes I just noticed it and pushed a change 😉